### PR TITLE
sys-apps/dstat: version bump to 0.7.3

### PR DESCRIPTION
**Pull request by henning-schild**
Signed-off-by: Henning Schild <henning@hennsch.de>

### CODE REVIEW

File: sys-apps/dstat/dstat-0.7.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RDEPEND="wifi? ( net-wireless/python-wifi )"
+DEPEND=""
+
+src_install() {
+	default
+	if use examples; then
+		dodoc examples/{mstat,read}.py
+	fi
+	if use doc; then
+		dohtml docs/*.html
```
Comment (doughdemon) [inline sys-apps/dstat/dstat-0.7.3.ebuild]: better use dodoc

**Comment by doughdemon** [comment]
@wschlich

**Comment by doughdemon** [comment]
@wschlich

**Comment by monsieurp** [comment]
Not sure he is active at the moment.

**Comment by heroxbd** [comment]
Hi Henning, thanks for the version bump.  Just to comment that your bump is broken on Gentoo Prefix, which is a regression from sys-apps/dstat-0.7.2-r1

**Comment by henning-schild** [comment]
Hi @heroxbd, thanks for the report. Is there more information available on that regression? Or did your commit 543b5ea6264543d34e8ea5ebc553e79012fa3c7e solve the problem and there is nothing to do for me?

**Pull request closed without merging**